packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -321,7 +321,7 @@ const AccountSwitcher = observer(({ history, is_mobile, is_visible }) => {
                         <div className='acc-switcher__separator' />
                     </React.Fragment>
                 ) : null}
-                {(!is_high_risk || is_eu) && has_maltainvest_account ? (
+                {(!is_high_risk && has_maltainvest_account) || is_eu ? (
```

Comment:
We shouldn't show the MF account creation button to the low risk client ( DIEL) who doesn't have MF account , so I adjusted the condition here we show if it is low risk and have mf account or it is eu account.